src/balloon.rs

```diff
@@ -0,0 +1,33 @@
+use volatile::access::{ReadOnly, ReadWrite};
+use volatile_macro::VolatileFieldAccess;
+
+pub use super::features::balloon::F;
+use crate::le32;
+
+/// Traditional Memory Balloon Device Configuration Layout
+///
+/// Use [`ConfigVolatileFieldAccess`] to work with this struct.
+#[doc(alias = "virtio_balloon_config")]
+#[cfg_attr(
+    feature = "zerocopy",
+    derive(
+        zerocopy_derive::KnownLayout,
+        zerocopy_derive::Immutable,
+        zerocopy_derive::FromBytes,
+    )
+)]
+#[derive(VolatileFieldAccess)]
+#[repr(C)]
+pub struct Config {
+    #[access(ReadOnly)] // TODO: Not actually specified by the spec?
```

Comment:
I think we can safely remove the TODO, right? Every time this field is mentioned in the spec, the driver reads it. It would also not make sense to make this driver-writable in the grand scheme of things.


```suggestion
    #[access(ReadOnly)]
```